custom_components/tuya_local/devices/haier_airmart_ac.yaml

```diff
@@ -0,0 +1,99 @@
+name: Haier / Airmart AC (Tuya 3.5)
+products: []
```

Comment:
products should be left out if empty, but it would be better to find your product id. You should be able to see it using `python -m tinytuya scan`, or in the HA log if adding to tuya-local using the cloud config flow.